packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -894,14 +894,24 @@
             }
         }
 
-        .proof-of-identity__submit-button {
-            @include desktop {
-                margin-top: 3.2rem;
-                margin-left: auto;
+        .proof-of-identity {
+            &__actions {
+                @include desktop {
+                    display: flex;
+                    justify-content: flex-end;
+                    align-items: flex-end;
+                }
             }
 
-            @include mobile {
-                margin-inline: 2.4rem;
+            &__submit-button {
+                @include desktop {
+                    margin-top: 3.2rem;
+                    margin-left: auto;
```

Comment:
Pls combine this. Lets avoid longhand properties

Reply:
I suppose it doesn't hurt keeping this single line separate, maybe if we've move than two